configfs/faketsm/fakereport.go

```diff
@@ -311,6 +312,49 @@ func checkV7(privlevelFloor uint) func(*ReportEntry, string, []byte) error {
 	}
 }
 
+func read611(privlevelFloor uint) func(*ReportEntry, string) ([]byte, error) {
```

Comment:
I might be missing contexts here, what's "611"?

Reply:
Oh it’s in a comment. This is the behavior as of Linux 6.11